src/styles/panels.less

```diff
@@ -16,6 +16,24 @@
     &.panel-open {
         .panel-heading {
             .ux-icon-down();
+            &:before {
+                transform: none;
+            }
+        }
+    }
+}
+
+/** RTLCSS cannot account for mixin changes so adding specific style to rotate the icon **/
+[dir="rtl"] {
+    .accordion-chevron {
+        .panel-heading {
+            .ux-icon-next();
+        }
+
+        &.panel-open {
+            .panel-heading {
+                .ux-icon-down();
+            }
```

Comment:
If you're planning to flip the next and previous icons, this can be removed.

Reply:
@donna-marijne because these were mixins vs class calls I couldn't use that approach here and ended up w/ this one, but since i can't edit ux-icons.less I need to think of another approach to swap next/previous and chevrons.